packages/py-api/py_api/main.py

```diff
@@ -1,10 +1,11 @@
-from dotenv import load_dotenv  # noqa
+from dotenv import load_dotenv
+from py_api.utilities.logging import get_log_config
+from py_api.utilities.parsers import eval_bool  # noqa
 
 load_dotenv()  # noqa
```

Comment:
Move this after the import of load_dotenv

Reply:
Done